src/components/Attachments/AttachmentCarousel/Pager/index.tsx

```diff
@@ -146,14 +150,11 @@ function AttachmentCarouselPager(
 
     const carouselItems = items.map((item, index) => (
         <View
-            key={item.source}
+            key={item.source as string}
             style={styles.flex1}
         >
             <CarouselItem
-                // @ts-expect-error TODO: Remove this once AttachmentView (https://github.com/Expensify/App/issues/25150) is migrated to TypeScript.
                 item={item}
-                isSingleItem={items.length === 1}
-                index={index}
```

Comment:
why those props were removed?

Reply:
It's no longer used by CarouselItem, probably smb forgot to remove this.